src/main/java/com/esri/samples/featurelayers/feature_collection_layer/README.md

```diff
@@ -0,0 +1,36 @@
+<h1>Feature Collection Layer</h1>
+
+<p>Demonstrates how to create a new Feature Collection with a Point, Polyline, and Polygon Feature Collection Table. The Feature Collection is then displayed on the ArcGISMap as a Layer.</p>
+
+<p><img src="FeatureCollectionLayer.PNG"/></p>
+
+<h2>How it works</h2>
+
+<p>To display a <code>FeatureCollection</code> as a <code>FeatureCollectionLayer</code> on an <code>ArcGISMap</code> using different <code>FeatureCollectionTable</code>s:</p>
+
+<ol>
+    <li>Create a feature collection layer using a new feature collection, <code>new FeatureCollectionLayer(FeatureCollection)</code></li>

```

Comment:
please use lowercase names for arguments as if you had a variable with that name -> `featureCollection`. Same for the other code blocks.